pegasus/sites.v3/code.org/public/minecraft.haml

```diff
@@ -1,16 +1,22 @@
 ---
+<%
+  mc_landing_2017 = !!DCDO.get('hoc_launch', {})['mc_landing_2017']
+  year = mc_landing_2017 ? 2017 : 2016
+  share = mc_landing_2017 ? I18n.t(:minecraft_og_agent) : I18n.t(:minecraft_og_designer)
+%>
+
 title: Minecraft
 theme: responsive
 social:
   "twitter:card": photo
   "twitter:site": "@codeorg"
   "twitter:url": "http://code.org/minecraft"
-  "twitter:image:src" : "https://code.org/images/mc/mc_social_2016.jpg"
+  "twitter:image:src" : "https://code.org/images/mc/mc_social_<%= year %>.jpg"
   "twitter:title": "Minecraft Hour of Code"
-  "twitter:description": "Minecraft is back with a new activity for the Hour of Code. Program animals and other Minecraft creatures in your own version of Minecraft."
+  "twitter:description": "<%= share %>"
   "og:title": "Minecraft Hour of Code"
-  "og:description": "Minecraft is back with a new activity for the Hour of Code. Program animals and other Minecraft creatures in your own version of Minecraft."
-  "og:image" : "https://code.org/images/mc/mc_social_2016.jpg"
+  "og:description": "<%= share %>"
+  "og:image" : "https://code.org/images/mc/mc_social_<%= year %>.jpg"
```

Comment:
nit: It feels a bit inconsistent that for image URL we substitute a year into a longer string, but for text we just have a single variable containing the final string.  But not a big deal.